mne/annotations.py

```diff
@@ -880,6 +910,29 @@ def _check_event_id(event_id, raw):
         raise ValueError('Invalid input event_id')
 
 
+def _check_event_description(event_desc, events):
+    """Check event_id and convert to default format."""
+    # check out event_id dict
+    if event_desc is None:  # convert to int to make typing-checks happy
+        event_desc = list(np.unique(events[:, 2]))
+
+    if isinstance(event_desc, np.ndarray):
+        if event_desc.ndim == 1:
+            event_desc = list(event_desc)
+
+    if isinstance(event_desc, dict):
+        for val in event_desc.values():
+            _validate_type(val, (str, None), 'Event names')
+    elif isinstance(event_desc, list):
+        event_desc = dict(zip(event_desc, (str(i) for i in event_desc)))
+    elif callable(event_desc):
+        pass
+    else:
+        raise ValueError('Invalid input event_id')
```

Comment:
why you raise an error say what you got and what would have been expected. Something like:
```suggestion
        raise ValueError('Invalid type for event_id (should be list, dict or callable). Got %s'  % type(event_id))
```